src/backends/libtorch/torch_backend_c2.cc

```diff
@@ -0,0 +1,322 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
```

Comment:
The "c2" in the filename refers to "caffe2" so it is not a good name here. Perhaps libtorch_backend_pt.cc.

The point of this file is that it will actually be compiled into caffe2 when we rebuild it and so it is the only file that has to know anything about caffe2 includes, datatypes, etc. This was needed because caffe2 didn't have an API that TRTIS could use directly. If the libtorch API can be used directly (specifically, it does not require any protobufs to be passed as parameters, and does not require any datatypes as parameters that aren't available in the libtorch header) then you don't need this isolation (it all depends on the libtorch API, so we can discuss if you are unsure).

Reply:
Workspace was not required for LibTorch. Refactored to fix this.